release.sh

```diff
@@ -81,9 +81,21 @@ git push origin "$TAG_NAME"
 
 echo "**** Pushing build $BUILD ****"
 
+rm -f $TARBALL.sig install.sh.sig
+
+# Sign the tarball and the install script. Note that we don't sign the channel build number because
+# it wouldn't accomplish anything: If an attacker wanted to provide an old number, they could
+# provide the old signature to match. If an attacker provided a number that hasn't been used
+# before, they would not be able to provide a matching package because no such signed package
+# exists.
+gpg -u 9DA8CCBD --detach-sig $TARBALL
```

Comment:
For this case, even if someone injected a dupe key into my keyring, the result would be pushing a good build with a bad signature, whereas an attacker probably would prefer to push a bad build with a good signature... :) But sure, I'll extend it.